ga4gh/backend.py

```diff
@@ -470,7 +472,15 @@ def searchDatasets(self, request):
             protocol.SearchDatasetsResponse,
             self.datasetsGenerator)
 
-    # Iterators over the data hieararchy
```

Comment:
This was an intentional separator between the different groups of methods. I'd like to keep this unless there's a good argument for removing it.